docs/openapi.md

```diff
@@ -0,0 +1,61 @@
+# OpenAPI description document
+
+`ucc-gen` command is executed with `--openapi` argument enabled by default.
+There have to be defined valid `globalConfi.json` and `app.manifest` to have OpenAPI description document (`static/openapi.json` file) generated.
+
+To disable OpenAPI functionality for `ucc-gen` run, call it with `--openapi=0` argument.
+
+## How to find the document?
+
+Once `ucc-gen` command is executed, OpenAPI description document is located in output `static` subdirectory.
+
+When add-on is installed to Splunk instance, it is exposed via web and management interface, so is available under following addresses accordingly:
+
+* \[protocol\]://\[domain\]:\[port\]/en-GB/splunkd/__raw/servicesNS/\[user\]/\[appname\]/static/openapi.json
+
+(eg. http://localhost:8000/en-GB/splunkd/__raw/servicesNS/admin/Splunk_TA_cisco_meraki/static/openapi.json)
```

Comment:
@artemrys  ,
Any idea how we can make it better?

Reply:
I think this is good enough.